stock-service/src/main/resources/application-dev.yml

```diff
@@ -34,6 +34,11 @@ spring:
               action: create
               create-target: /dev/stderr
 
+  config:
+    import: classpath:env.properties
```

Comment:
we should be passing this via the toplevel `.env` and into passing it via `--property.name=propertyValue` flags in the compose file